### PR TITLE
Portfolio desktop version review

### PR DESCRIPTION
**Pull request by hagan199**
#Portfolio desktop version. 
To make my portfolio responsive and to adopt to mobile framework and desktop as well.
1. Added contact-form.
2. Added contact  style.css.
3. Fix Linter error. 
4.  Added submit to be hover.
5. Update index.html;

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1,276 +1,398 @@
...
+body {
+  background-color: var(--white);
+  width: 100%;
+  font-family: var(--font-family-style);
+}
+
+.top-mob {
+  height: 718px;
+  background-image: url('/assets/background-desk.png');
+  background-color: rgb(252, 246, 246);
```
Comment (Omar-Muhamad) [inline assets/css/style.css:10]: - [ ] Kindly add the correct path for the background image here to make it work online for this situation you can make it like that: `background-image: url('../background-desk.png');`
- [ ] Please remove the background color or make it white to match the Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image  |

File: assets/css/style.css [hunk 2]
```diff
@@ -1,276 +1,398 @@
...
-.card-image {
+.work-inside {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
   width: 90%;
-  margin-top: 1.5rem;
+  list-style: none;
+  padding: 8rem 1.6rem;
+  row-gap: 5em;
+  justify-content: center;
```
Comment (Omar-Muhamad) [inline assets/css/style.css:133]: - [ ] Please add `margin: 0 auto;` after your width property to center the work cards to match the Figma design.
- [ ] Please increase the `row-gap` value to match the Figma design and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image  |

File: assets/css/style.css [hunk 3]
```diff
@@ -1,276 +1,398 @@
...
 }
 
-.language-section {
-  margin-top: 3rem;
+.user-tool > div {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
 }
```
Comment (Omar-Muhamad) [inline assets/css/style.css:272]: - [ ] Please add the correct font size, font-weight, and margins to these heading text and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  |image |

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @hagan199,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Ismailco** [comment]
@hagan199,
I am reviewing your PR, normally I would invalidate it, but since the first reviewer give you a review I need to review it, please fix the linter errors and add a description to the PR as soon as possible so I can re-review your PR.

**Review by Ismailco — COMMENTED** [review]
Hi @hagan199,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not pass the linters and has no description in the PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: assets/css/style.css [hunk 4]
```diff
@@ -1,276 +1,428 @@
-.header-container {
+body {
+  background-color: var(--white);
```
Comment (Ismailco) [inline assets/css/style.css:2]: - [ ] [optional] Kindly change the background color of the body to ``` #E5E5E5 ``` so your design can be like the Figma one

Screenshot_20220506_171133

File: assets/css/style.css [hunk 5]
```diff
@@ -1,276 +1,428 @@
...
 }
 
-.about-button {
-  padding: 1.5rem 2.2rem;
-  margin-top: 3rem;
-  border-radius: 10px;
-  border: 1px solid #396df2;
-  color: #396df2;
-  font-size: 1.4rem;
+.user-tool-d {
```
Comment (Ismailco) [inline assets/css/style.css:283]: - [ ] As the previous reviewer suggests, please use the correct font size, and font-weight, for the ```Languages```, ```Frameworks```, ```Skills```  and try to stick as much as possible to the given Figma design.

Screenshot_20220506_171033

File: assets/css/style.css [hunk 6]
```diff
@@ -1,276 +1,428 @@
...
+  background-color: var(--white);
+  width: 100%;
+  font-family: var(--font-family-style);
+  position: relative;
+}
+
+.top-mob {
+  height: 718px;
+  background-image: url('/assets/background-desk.png');
+  background-size: cover;
```
Comment (Ismailco) [inline assets/css/style.css:11]: - [ ] [optional] You can use the ``` background-size: contain ``` to replicat the background from the Figma design

 - Figma Design:
 image

- Your Design:
image

File: assets/css/style.css [hunk 7]
```diff
@@ -1,276 +1,428 @@
...
+  color: #6070ff;
+  font-weight: 500;
 }
 
-.btn {
-  padding: 1.5rem 2.2rem;
-  margin-top: 3rem;
-  border-radius: 10px;
-  border: 1px solid #396df2;
+.btn-in {
```
Comment (Ismailco) [inline assets/css/style.css:229]: - [ ] [optional] The elements of the form should be centered on the page including the ``` Get in touch``` button, please fix it

-Your design:
Screenshot_20220506_174115

-Figma design:
Screenshot_20220506_174314

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @hagan199,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding a descriptive pull request message to your pull request to describe changes made in your  pull request

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @hagan199 ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
-No linter errors :+1: 
- Good use of GitHub flow :+1: 
- The app is working fine :+1: 

## Optional suggestions :small_airplane: 
- [OPTIONAL] Kindly make sure to use the current microverse README [template](https://github.com/microverseinc/readme-template) as required in the project.
- [OPTIONAL] In my opinion, it would be better to write a descriptive PR summary as it would make your project more professional
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @hagan199 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to the [Microverse README template](https://github.com/microverseinc/readme-template) which will help you to keep your repo clean and more understandable. Please make sure to customize it according to your portfolio.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**